doc/api/deprecations.md

```diff
@@ -162,9 +162,10 @@ explicitly via error event handlers set on the domain instead.
 <a id="DEP0013"></a>
 ### DEP0013: fs asynchronous function without callback
 
-Type: Runtime
+Type: End-of-Life
 
-Calling an asynchronous function without a callback is deprecated.
+Calling an asynchronous function without a callback will throw a `TypeError`
```

Comment:
I think this can be present tense: "... without a callback throws a `TypeError` from Node XX onwards."